src/patternfly/components/Button/examples/buttonStates.hbs

```diff
@@ -1,3 +1,58 @@
-{{#> button btnClass="pf-is-primary pf-has-focus"}}Focused{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-active" btnAriaPressed="true"}}Active{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-disabled" btnTabIndex="-1" btnAriaDisabled="true"}}Disabled{{/button}}
\ No newline at end of file
+<div class="pf-l-level">
+<div class="pf-l-stack">
+    {{#> button btnClass="pf-is-primary"}}
+        Primary
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-focus"}}
+        Primary Focus
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-active"}}
+        Primary Active
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-disabled" btnAttributes="disabled=\"true\" " }}
```

Comment:
HTML's `disabled` attribute needs a little more than just being set to function as expected with respect to accessibility. Because it's a boolean attribute [1], simply having the attribute present, even without a value assigned at all, will have some effect depending on your combination of OS and screen reader. The problem is that support will be unstable across a variety of user agent, browsers, etc.

For best support, consider using `disabled="disabled"`. In this case, VoiceOver properly announced the "dimmed" state when it came across it.

[1] https://www.w3.org/TR/html5/infrastructure.html#boolean-attributes

Looks great otherwise, nice work!

Reply:
OK, I've made this change. Thanks, @seanforyou23 !